.gitignore

```diff
@@ -1,2 +1,2 @@
-front_end/node_modules/**
-front_end/.eslintcache
\ No newline at end of file
+/node_modules/**
+dist/
```

Comment:
`bin/` might be handy to include as well

Reply:
Will do!